humble/distribution.yaml

```diff
@@ -155,6 +155,21 @@ repositories:
       url: https://github.com/ros2/ament_cmake_ros.git
       version: humble
     status: maintained
+  ament_download:
+    doc:
+      type: git
+      url: https://github.com/samsung-ros/ament_download.git
```

Comment:
@clalancette is it fine for this to use the `ament` namespace? (we already merged it on dashing)